README.md

```diff
@@ -1,61 +1,122 @@
+
+> Behave as if you were at a professional event
+
+Although we are on the internet - and perhaps even more importantly _because_ we are on the internet where tone is extremely challenging to read - choosing our words carefully and conducting yourself as a professional is important. Not sure if what you're about to say is professional? Try imagining someone in your family or friend group who's got your back, but holds you to high standards - whether that be your grandma, your cousin, our best friend or your mom - would they be cheering on what you're about to say, or would they be disappointed in your level of professionalism? 
+
+> Use kindness to build community
+
+This doesn't mean we don't hold each other to high standards for behaviour, but it does mean we call each other in (or out, if necessary) on things we are uncomfortable with in a way that is grounded in empathy and respect - even if we don't feel like we ourselves were respected in that moment. 
 
-* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
-* Exercise consideration and respect in your speech and actions. Tone on the internet is often misunderstood, and people may interpret your words differently than you might intend. So err on the side of caution.
+> Be curious to hear about the other person’s perspective
+
+Remember not to assume your experience is the default - make space for others to have different life experiences than you. One litmus test for how you are framing something is going to read as having implicit bias is to check against these three questions: 
+* *First: is this true* is what you're about to say something that is a fact, or a perspective that you hold where you can acknolwedge that others may hold different ones? Is there a way to explicitly acknolwedge that you're open to other perpsectives in how you're responding; will this read as a "yes and", or a "no because"? 
+* *Second: Is this missing something* are you encouraging all possibilities, or just the ones that come most readily to you and your life experience? What questions might you ask the other folks in the conversation to understand their experiences or motivations better? 
+* *Third: is this just* is this supporting the inclusion and equality of minority members of our community, or reinforcing barriers for those groups? This can be quite nuanced, but being explicit about who and how we want to advocate for each other, and why it matters to us, is always a great reminder. 
+
+Take the time to understand the motivations behind what the other person has share, rather than drawing your own conclusions. When in doubt: repeat back what you think you're hearing, and ask why the topic matters to them. 
```

Comment:
typo:

```suggestion
Take the time to understand the motivations behind what the other person has shared, rather than drawing your own conclusions. When in doubt: repeat back what you think you're hearing, and ask why the topic matters to them. 
```